openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -76,11 +76,43 @@ OpenAssessment.ItemUtilities = {
  OpenAssessment.Prompt
  **/
 OpenAssessment.Prompt = function(element, notifier) {
+    if (this.tinyMCEEnabled) {
+        element = this.attachWysiwygToPrompt(element);
+    }
     this.element = element;
     this.notifier = notifier;
 };
 
 OpenAssessment.Prompt.prototype = {
+    tinyMCEEnabled: window.tinyMCE !== undefined,
+
+    /**
+     Attach Wysiwyg editor to the textarea field.
+
+     Args:
+     el (OpenAssessment.Container): The container that the prompt is a member of.
+
+     Returns:
+     Updated OpenAssessment.Container
+
+     **/
+    attachWysiwygToPrompt: function(el) {
+        var elId = $(el).find('textarea').attr('id');
+        if (!elId) {
+            /* jshint undef:false */
+            var textarea = $(el).find('textarea');
+            var text = $(textarea).val();
+            var type = $(textarea).data('type');
+            if (text && (type === 'text')) {
+                text = _.escape(text).replace(/(?:\r\n|\r|\n)/g, '<br />');
+                $(textarea).val(text);
+            }
+            var newElId = Date.now() + '-textarea-' + (Math.random() * 100);
+            $(textarea).attr('id', newElId).tinymce(tinymceCfg());
```

Comment:
Not sure where in the code it would be set, but would be nice to give the html editor focus when it's created so the cursor starts in there. Especially when you create a new prompt the focus should be automatically in the html editor. Not a blocker but would be nice

Reply:
added `focus()` after create element